regression_em/model.py

```diff
@@ -1,18 +1,209 @@
-import scipy.sparse as sp
-from sklearn.linear_model import LinearRegression
+"""Package to perform regression EM algorithmself.
+For the detail of this algortihm,
+see [https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/46485.pdf].
+"""
+from random import random
+from typing import Sequence, Tuple
 
+import numpy as np
+from sklearn.linear_model import LinearRegression, Ridge
 
-class RegressionEm:
-    def fit(self):
-        pass
+# Define type.
+RegParam = Tuple[np.ndarray, float]
 
-    def predict(self):
-        return 1
 
+class RegressionEM:
```

Comment:
Inherit BaseEstimater of scikit-learn. 
https://github.com/scikit-learn/scikit-learn/blob/b194674c42d54b26137a456c510c5fdba1ba23e0/sklearn/base.py#L136